mitodl/odl-video-service
# Fix: add nubbins for celery monitoring

This commit adds two Celery client configurables necessary for monitoring.

    CELERY_TASK_TRACK_STARTED
    CELERY_TASK_SEND_SENT_EVENT

What are the relevant tickets?

For: mitodl/ol-infrastructure#2290

### What are the relevant tickets?
For: mitodl/ol-infrastructure#2290

### Description (What does it do?)
This commit adds two Celery client configurables necessary for [celery monitoring](https://celery-monitoring.odl.mit.edu).

    CELERY_TASK_TRACK_STARTED
    CELERY_TASK_SEND_SENT_EVENT


### How can this be tested?

Log on to [celery monitoring](https://celery-monitoring.odl.mit.edu). Check odl-video and observe task status.
